cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1260,6 +1267,19 @@ protected String getServiceAccountName() {
                 if (listeners.getPlain().getAuthentication() != null) {
                     varList.add(buildEnvVar(ENV_VAR_KAFKA_CLIENT_AUTHENTICATION, listeners.getPlain().getAuthentication().getType()));
                 }
+
+                if (listeners.getPlain().getAuthentication() instanceof KafkaListenerAuthenticationSslPlaintextVault) {
```

Comment:
This is good. But again - similarly to the scripts in the Docker image we have to do this 3 times:
* For the unecnrpyted interface
* For the TLS interface
* For the external interfce